social-network/src/creatify/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import yourArticles from './your-articles.json';
+import '../App.css';
+
+class Creativityarticle extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      articles: yourArticles
+  }
+  }
+    render() {
+      let articles = this.state.articles.map((article, idx) => {
+        return(
```

Comment:
I would've made a component representing a single card and used it in lines 14-25 here and lines 16-26 in missed-articles.js